exercises/phone-number/src/test/java/PhoneNumberTest.java

```diff
@@ -25,6 +25,7 @@ public void cleansTheNumber() {
         assertEquals(
                 expectedNumber, actualNumber
         );
+        
```

Comment:
I don't think this extra empty line is needed? Not very important of course, but it would be great if you could remove it if you have time 🙂

Reply:
done